R/generate_virtual_pop.R

```diff
@@ -13,6 +13,23 @@
 #'@examples
 #'generate_virtual_pop(100, "height", rnorm, 0, 1)
 #'generate_virtual_pop(100, "n_accidents_per_hr", rpois, 2)
+#'
+#' @importFrom rlang :=
+#' @export
 generate_virtual_pop <- function(N, var_name, dist, ... ){
```

Comment:
do you think N input needs to be tested? I tried it with decimal values and it rounds down and takes that many samples... what do you think?

Reply:
also comments through out code? eg. what  exceptions are checking for

Reply:
> do you think N input needs to be tested? I tried it with decimal values and it rounds down and takes that many samples... what do you think?

I think you are correct! I opened an issue addressing this. #39

Reply:
> do you think N input needs to be tested? I tried it with decimal values and it rounds down and takes that many samples... what do you think?

That's definitely a good idea. Addressed #41